pages/common/yesod.md

```diff
@@ -0,0 +1,19 @@
+# yesod
+
+> Yesod web framework CLI helper.
+
+- Install Yesod CLI tool in a Yesod scaffolding:
```

Comment:
Totally agree, however this doesn't refer to the installation of the tool, but rather the instantiation of the CLI in the scaffolding so it can be invoked (this is performed constantly, even on frequent updates of the underlying libraries being used in the project ), the term `installation` here allows that ambiguity in the context, but is the accepted term and well understood for the community.

Moreover the *installation* (instantiation) is both the most vital and the most forgettable part of the CLI usage, actually it constitutes the main reason for this *TLDR* page. Anyways if it is a major impediment I will remove it.